setup/install/providers/aws/aws-ec2.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: single
+title:  "Amazon EC2"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+In [AWS](https://aws.amazon.com/){:target="\_blank"}, an [__Account__](/concepts/providers/#accounts)
+maps to a credential able to authenticate against a given [AWS
+account](https://aws.amazon.com/account/){:target="\_blank"}.
+
+## Option-1 : Use AWS Console to configure AWS
+
+Use this option to deploy Spinnaker, if you are familar with deployment using [AWS Console](https://console.aws.amazon.com/) .
+
+### Managing Account
+1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
+2. Download [the template](https://github.com/spinnaker.github.io/blob/master/setup/install/providers/aws/managing.yaml) locally to your workstation.
```

Comment:
Doesn't need to change in this PR, but I think this might be better placed in another repository, but I don't believe a great place for it exists yet.